etna/packages/etna-js/spec/helpers.js

```diff
@@ -11,15 +11,34 @@ export const stubUrl = ({
   request,
   status = 200,
   headers = {},
-  host = 'http://www.fake.com'
+  host = 'http://localhost'
 }) => {
-  nock(host)
-    [verb](path, request)
-    .reply(status, response, {
-      'Access-Control-Allow-Origin': '*',
-      'Content-type': 'application/json',
-      ...headers
-    });
+  let nocked;
+
+  return new Promise((resolve, reject) => {
```

Comment:
`nock` by default doesn't have a sane way to order request mocks, which is important if you want to be certain that requests come in a particular order.

This minor change makes the return value of this function more useful; it's a promise that is only fulfilled when the given mocked response is actually served, allowing one to use promises to chain the order together in which you expect requests to come.  Doesn't break any existing usages.